<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="0">
     <widget class="QLabel" name="labelWeather">
      <property name="text">
       <string>labelWeather</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelTemp">
      <property name="text">
       <string>labelTemp</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="labelHumidity">
      <property name="text">
       <string>labelHumidity</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="btnExportStarMap">
      <property name="text">
       <string>生成灵感星图</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCalendarWidget" name="calendarWidget"/>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="btnHistoryEvent">
      <property name="text">
       <string>历史上的今天</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>今日灵感三连</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="labelWind">
      <property name="text">
       <string>labelWind</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
